>From 04b7d919458c9c4ecabbf40a793d5d22ae41b686 Mon Sep 17 00:00:00 2001
From: "Marcin Kulisz (kuLa)" <debian@kulisz.net>
Date: Fri, 22 Nov 2013 19:38:52 +0000
Subject: [PATCH] Allowing Awesome to be listed on sessions managers+

list in logging window of a display manager like lightdm
---
 awesome.desktop | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/awesome.desktop b/awesome.desktop
index 1f7878c..6f830b9 100644
--- a/awesome.desktop
+++ b/awesome.desktop
@@ -2,7 +2,7 @@
 Encoding=UTF-8
 Name=awesome
 Comment=Highly configurable framework window manager
-NoDisplay=true
+NoDisplay=false
 TryExec=awesome
 Exec=awesome
 Type=Application
-- 
1.8.4.3

